libs/clients/passports/src/lib/passports.config.ts

```diff
@@ -17,7 +17,7 @@ export const PassportsClientConfig = defineConfig<z.infer<typeof schema>>({
     return {
       xRoadServicePath: env.required(
         'XROAD_PASSPORT_LICENSE_PATH',
-        'IS-DEV/GOV/10001/SKRA-Protected/Forskraning-V1',
```

Comment:
Is this change relevant to all services using this client?

Reply:
Yes, they have updated their x-road for both passports and id-cards so this is relevant to all